### DISCUSSION TITLE
IP Pools, Ip addres no rotating

### DISCUSSION THREAD

**Discussion by arunkuttytry** [postalserver/postal#2120] [Help with using Postal]
Hi 
i am done with ip pools and assign org but, it send only for main ip not rotating, can you please help me

**Comment by willpower232** [comment]
Can you confirm with `ip a` that all the IP addresses you've put in your pool(s) are available on the server?

**Reply by arunkuttytry** [reply]
same interface main ip sub ip also but it send only main ip
